workflow-templates/im-deploy-on-prem-database.yml

```diff
@@ -177,6 +180,32 @@ jobs:
           DB_SERVER_NAME@stage demo: ''
           DB_SERVER_NAME@prod: ''
 
+      - name: AZ Login
+        uses: azure/login@v1
+        with:
+          # TODO: These the Service Principal these credentials are for needs access to Vault. See this git-er-done article for more information:
```

Comment:
This might need some re-wording.

Reply:
...did I have a stroke while I wrote this?!